SU-HSDO/suhumsci
# STN-793:  Bug fix aligned images in wysiwyg

# READY FOR REVIEW

## Summary
When inserting an image in the Text Editor, and the image is set to align left or right, the CKEditor does not closely reflect the correct styles. (Note: this only effects the editor preview, not the final page.

JIRA Card: https://sparkbox.atlassian.net/browse/STN-793

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. npm run test
2. `lando drush @sparkbox_sandbox.local cr` - this work adds a new media image field template and needs a cache clear to pick it up.
2. Open a page with images in WYSIWYG or create a new one.
3. Add images in through the WYSIWYG media button, when adding those images, use different scales and sizes.
4. Align the images left, right and center, with text below the image that aligns to the left or right of the image if you choose an alignment for the image. These should float correctly next to each other unless the image is centered.
5. The images should align correctly left and right and should also take up the correct amount of space to more closely reflect what the alignment with text will look like when saved.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
